rfcs/20240503-opset-deprecations.md

```diff
@@ -0,0 +1,102 @@
+# RFC: StableHLO v1.0 Opset Deprecations & Cleanups
+
+Author: gleasonk<br/>
+Last Modified: 5/3/24<br/>
+Status: In review<br/>
+
+## Background
+
+This doc covers a list of opset cleanups that we want to do for StableHLO v1.0.
+Most of these ops were never spec’ed and therefore have no formal compatibility
+guarantees, per the Unspecced Features compatibility exemption, however we can
```

Comment:
Add a link? https://github.com/openxla/stablehlo/blob/main/docs/compatibility.md#out-of-scope